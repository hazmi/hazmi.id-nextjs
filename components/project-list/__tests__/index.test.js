import React from 'react'
import renderer from 'react-test-renderer'
import ProjectList from '../index'

describe('With Snapshot Testing', () => {
  it('show the correct content', () => {
    const component = renderer.create(<ProjectList />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})