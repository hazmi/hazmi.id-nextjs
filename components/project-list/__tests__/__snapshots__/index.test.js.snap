// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing show the correct content 1`] = `
<div
  className="project-list"
>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      Connect33
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="Connect33"
        src="/static/project/connect33.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        A HR mobile application.
      </p>
    </div>
  </div>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      MUTUCare
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="MUTUCare"
        src="/static/project/mutucare.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        A hazard awareness mobile application.
      </p>
    </div>
  </div>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      CelliHealth
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="CelliHealth"
        src="/static/project/cellihealth.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        A healthcare mobile application.
      </p>
    </div>
  </div>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      IN2NITE
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="IN2NITE"
        src="/static/project/in2nite.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        A last minute hotel booking mobile application.
      </p>
    </div>
  </div>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      Kitook
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="Kitook"
        src="/static/project/kitook.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        An Online Travel Agent (OTA) website.
      </p>
    </div>
  </div>
</div>
`;
